app/views/helper/GlobalSection.scala

```diff
@@ -19,25 +19,30 @@ package views.helper
 object GlobalSection {
 
   import uk.gov.hmrc.thirdpartydeveloperfrontend.controllers.FormKeys._
+  import uk.gov.hmrc.thirdpartydeveloperfrontend.controllers.FieldNameKey
+  import uk.gov.hmrc.thirdpartydeveloperfrontend.controllers.GlobalMessageKey
 
-  private def a(em: String)(found: String => String, notFound: () => String) = {
-    (globalKeys.contains(em), globalToField.get(em)) match {
-      case (true, Some(s)) => found(s)
-      case _               => notFound()
-    }
+  private def a(rawErrorMessageKeyOrMessageText: String)(found: FieldNameKey => String, notFound: () => String) = {
```

Comment:
Is there a better name for this method than `a`?

Reply:
No, it's an HTML anchor, so it's an a

Reply:
I think it was named for anchor in html which it was probably used for originally but we haven't changed that.

Reply:
Just the types inbound changed

Reply:
It would likely become fn or on...